framework/validators/EachValidator.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\validators;
+
+use yii\base\InvalidConfigException;
+use Yii;
+use yii\base\Model;
+
+/**
+ * EachValidator serves validation of the array attributes.
+ * It perform validation of each array element using any other validator specified by [[rule]].
+ *
+ * ~~~php
+ * class MyModel extends Model
+ * {
+ *     public $arrayAttribute = [];
+ *
+ *     public function rules()
+ *     {
+ *         return [
+ *             ['arrayAttribute', 'each', 'rule' => ['trim']],
+ *             ['arrayAttribute', 'each', 'rule' => ['integer']],
+ *         ]
+ *     }
+ * }
+ * ~~~
+ *
+ * Note: this validator will not work with validation declared via model inline method.
```

Comment:
That's technical reason which could be solved. Inability to use inline method in this case doesn't sound good overall.